src/main/scala/nl.knaw.dans.easy.solr/Command.scala

```diff
@@ -15,87 +15,86 @@
  */
 package nl.knaw.dans.easy.solr
 
-import java.io.{File, FileInputStream}
-import java.lang.Thread._
+import java.io.{ File, FileInputStream }
+import java.lang.Thread.sleep
 
-import com.yourmediashelf.fedora.client.FedoraClient._
-import nl.knaw.dans.easy.solr.Defaults.filterDefaultOptions
-import org.apache.commons.configuration.PropertiesConfiguration
+import com.yourmediashelf.fedora.client.FedoraClient.findObjects
+import nl.knaw.dans.lib.error._
+import nl.knaw.dans.lib.logging.DebugEnhancedLogging
 import org.apache.commons.io.IOUtils.readLines
-import org.slf4j.LoggerFactory
+import resource._
 
 import scala.annotation.tailrec
 import scala.collection.JavaConverters._
-import scala.util.{Failure, Success, Try}
+import scala.language.reflectiveCalls
+import scala.util.control.NonFatal
+import scala.util.{ Failure, Success, Try }
 import scala.xml.PrettyPrinter
 
-object EasyUpdateSolrIndex {
+object Command extends App with DebugEnhancedLogging {
+  type FeedBackMessage = String
 
-  private val log = LoggerFactory.getLogger(getClass)
+  val configuration = Configuration()
 
-  /** API for EasyIngestFlow, assumes a single dataset id, fails on any type of error */
-  def run(implicit settings: Settings): Try[Unit] = Try {
-      settings.solr.update("<add>"+createSolrDoc(settings.datasets.head)+"</add>")
-      log.info(s"Committed ${settings.datasets.head} to SOLR index")
+  val commandLine: CommandLineOptions = new CommandLineOptions(args, configuration) {
+    verify()
   }
+  val app = new EasyUpdateSolrIndexApp(new ApplicationWiring(configuration))
```

Comment:
Since you are not using `EasyUpdateSolrIndexApp` to do any real work, it doesn't really make sense to keep it around (initializating it - which does nothing, automatically closing it - which does nothing)